doc/breadthFirstSearch/pgr_binaryBreadthFirstSearch.rst

```diff
@@ -221,11 +228,9 @@ Return Columns
     :end-before: return_path_end
 
 
-Example Data
+Additional Examples 
```

Comment:
Now that you are on it, use the sample data of the documentation.
To do that you need to use `edge_table`.
You will have to modify docqueries/breadthFirstSearch/doc-pgr_binaryBreadthFirstSearch.test.sql
Lets leave this for a next PR. (aka do not do it on this one)
Mention in the #2244 issue that pgr_binaryBreadthFirstSearch needs a revision of the examples used

Reply:
Never mind I see you already did something about it